src/content/docs/change-tracking/change-tracking-introduction.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Capture and analyze deployments and other changes'
+title: 'Capture and analyze recent changes in your systems'
```

Comment:
```suggestion
title: 'Capture and analyze changes in your systems'
```

I wouldn't state "recent" here because we can show arbitrarily old change events, subject to the customer's data retention for the `Deployment` event type in NRDB.